rna/README.md

```diff
@@ -0,0 +1,46 @@
+The workflows in this repository can be run by Cromwell standalone or in the Terra environment.
+The following section describes their use in the Terra environment:
+
+Metadata about samples can be stored in Terra workspace data
+(see https://support.terra.bio/hc/en-us/articles/360025758392-Populating-the-Data-table),
+which can then be used to drive workflows. When workflows complete successfully,
+Terra can update the `sample` data table (also known as `sample` "entities") with the location of their outputs.
+
+Thus if you set each workflow to run from the `sample` entity and set the `outputs` to update the `sample` entity, you can run a progression of workflows such as:
+
+1- Run `update-samples-entity` workflow. The `samples` table would then contain, for each sample:
```

Comment:
Sorry please update this such that step 1 is actually to upload a list of samples
which could be as simple as a 1-column CSV file:

```
entity:sample_id
SAMPLE_ID_1
SAMPLE_ID_2
SAMPLE_ID_3
```